### PR TITLE
FindFactorial

### PR DESCRIPTION
**Pull request by zeenayouhan**

### CODE REVIEW

File: factorial.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import java.util.*;
+public class factorial{
```
Comment (anku580) [inline factorial.java]: Please Rename the class name and file name as well to `Factorial`. The initial letter should be in caps for the Java class.

**Review by anku580 — CHANGES_REQUESTED** [review]
Please Rename the class name and file name as well to Factorial. The initial letter should be in caps for the Java class.

File: factorial.java (hunk 1 above)
Comment (zeenayouhan) [reply factorial.java]: Updated

**Review by anku580 — APPROVED** [review]

File: factorial.java (hunk 1 above)
Comment (anku580) [reply factorial.java]: Thank you 👍

**Pull request merged**